### ISSUE TITLE
Automated versioning only increments the minor version (the default)

### ISSUE THREAD

**Issue by gmyenni** [weecology/PortalData#224]
ie, [major], [patch] and [no version bump] tags don't work

**Comment by ethanwhite** [comment]
This works correctly for me locally. Can you run the following to confirm:

```
git checkout -b testbranch
git reset --hard 3838d6250864f6e2aece41488b800ef2ce3d6c9d
Rscript update_version.R
```

When I do this the version doesn't update.

**Comment by gmyenni** [comment]
Yep, can confirm that it doesn't update locally for me either.

**Comment by ethanwhite** [comment]
TRAVIIIIISSSSS!!!

Thanks. This will take some more serious debugging then. It might be a couple of days before I can get to it.

**Comment by ethanwhite** [comment]
I tried a fix for this in #281, but that doesn't seem to have solved the issue. This seems to be working in `livedat` so I'm not sure what's going on at the moment.

**Comment by ethanwhite** [comment]
This turned out to be primarily an issue with `git2r` not working with shallow branches and was fixed by #288. There was also a small additional issues fixed in 916bccb.
